### ISSUE TITLE
How to train on one GPU

### ISSUE THREAD

**Issue by cxq1** [ucas-vg/P2BNet#6]
I set the batch size to 8 on one gpu, how should I set the learning rate
When I set the learning rate to 0.01, the ap value on coco2017 is only 20.0

**Comment by Darren-pfchen** [comment]
batch size 8 for one GPU is high beyond the memory, and one GPU is time-consuming. We didn't try to do this on one GPU. If you  implement it on one stage, you can report it for me. Thank you!
